### ISSUE TITLE
How do I use just the UI for the queue and use my service for uploading?

### ISSUE THREAD

**Issue by ralphnico** [nervgh/angular-file-upload#808]
I am trying to use the queue functionality and then send the files on the FileItem to my own JavaScript service which will then use an API.

Any help is appreciated thanks!

**Comment by theo4u** [comment]
oh @ralphnico what you are trying to do, is to intercept the upload process and passing in your function to handle the main upload right ?

**Comment by ralphnico** [comment]
yes, so I want to be able to grab the files from the queue, and pass it to my own function that will send the actual HTTP request

**Comment by gunjan4455** [comment]
Hi @ralphnico 
I am facing the same issue. Have you find any solution for this?

**Comment by ralphnico** [comment]
@gunjan4455 what exactly is your issue?
